main.py

```diff
@@ -354,7 +409,7 @@ def move_organism(self, _organism, dx, dy):
         x, y = _organism.x + dx, _organism.y + dy
 
         self.remove_from_cell(_organism)
-        _organism.metabolize()
+        _organism.metabolize(dx + dy)
```

Comment:
I previously changed this line because it has a bug where if dx or dy is negative, the organism can gain energy instead of losing energy.

Reply:
I think you accidentally undid the changes I made in my PR to restrict the organism movement to one cell.